docgen/src/guide/Autocomplete_menu.md

```diff
@@ -6,17 +6,17 @@ category: guide
 navWeight: 20
 ---
 
-You can build with React InstantSearch an autocomplete menu that uses an external autocomplete component.
+You can build with React InstantSearch an autocomplete menu that uses an external autocomplete component. It takes advantage of the [`connectAutoComplete`](connectors/connectAutoComplete.html) connector. 
 
 [Read the examples](https://github.com/algolia/react-instantsearch/tree/master/packages/react-instantsearch/examples/autocomplete)
- showing you how to build an autocomplete menu that will:
-* display hits and facet values from the same index
+ showing you how to build an autocomplete menu that will let you:
 * display hits from different indices
+* have a textarea able to propose suggestions like mentions. 
 
 Those examples use advanced concepts from React InstantSearch. If you're not familiar with
 them you can read their guide first:
 
-* [Custom connectors](guide/Custom_connectors.html)
+* [Connectors](guide/Connectors.html)
 * [Virtual widgets](guide/Virtual_widgets.html)
```

Comment:
Well, `connectAutoComplete` is a connector? Before it, you needed to use custom connectors.